src/transformers/tf_utils.py

```diff
@@ -166,3 +166,78 @@ def check_embeddings_within_bounds(tensor: tf.Tensor, embed_dim: int, tensor_nam
             f"layer's input dimension ({embed_dim}). The likely cause is some problem at tokenization time."
         ),
     )
+
+
+def save_attributes_to_hdf5_group(group, name, data):
```

Comment:
These two functions are also moving targets I was trying to import from Keras - I've just given up and copied them into the transformers codebase.